Features/MyProfileLanguages.feature

```diff
@@ -0,0 +1,38 @@
+Feature: This test suite runs my profile language related test scenarios
+
+Scenario: 1. Verify user is able to add known languages with proficiency level
+	Given User logs into Mars application
+	When User adds 'English' language with 'Fluent' proficiency level into the known languages list
+	Then The 'English' language with 'Fluent' proficiency level should appear in the known languages list
+
+Scenario: 2. Verify user is able to edit an existing known language with proficiency level
+	Given User logs into Mars application
+	When User edits the 'English' language with 'Basic' proficiency level from the known languages list
+	Then The proficiency level for 'English' language should be updated to 'Basic'
+
```

Comment:
Please increase your coverage around the features you  are testing here is an idea of what is missing 

• Basic positive tests (happy paths)
• Extended positive testing with optional parameters
• Negative testing with valid input
• Negative testing with invalid input
• Destructive testing
• Security, authorisation, and permission tests (which are out of the scope of this post)

E.g of above scenarios

Happy path tests check basic functionality and the acceptance criteria of the API. We later extend positive tests to include optional parameters and extra functionality.

The next group of tests is negative testing where we expect the application to gracefully handle problem scenarios with both valid user input (for example, trying to add an existing username) and invalid user input (trying to add a username which is null).

Destructive testing is a deeper form of negative testing where we intentionally attempt to break the API to check its robustness (for example, sending a huge payload body in an attempt to overflow the system).

If some scenarios does not make sense to add you can you should know why it does not make  sense to add.